src/Agent.Listener/DistributedTask.Pipelines/Yaml/Contracts/Phase.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Microsoft.TeamFoundation.DistributedTask.Orchestration.Server.Pipelines.Yaml.Contracts
+{
+    internal class Phase : IPhase
+    {
+        internal String Condition { get; set; }
+
+        internal String ContinueOnError { get; set; }
+
+        internal IList<String> DependsOn { get; set; }
+
+        internal String EnableAccessToken { get; set; }
```

Comment:
This seemed appropriate to add to the phase. I need to check if this works when I plug this code into the server side (can't test this properly via local run).